demo/index.tsx

```diff
@@ -19,24 +19,40 @@ export default class Demo extends React.Component {
     introspection: defaultPreset,
   };
 
+  // not part of state because we parse it in constructor and just use in render
+  displayOptions = {};
+
   constructor(props) {
     super(props);
 
-    const { url, withCredentials } = getQueryParams();
+    const { url, withCredentials, ...restQueryParams } = getQueryParams();
+    // parse rest of query params we expect them to be in JSON format
+    // we might pick also some options that are not valid displayOptions but they will be ignored
+    // by Voyager component
+    for (const [key, value] of Object.entries(restQueryParams)) {
+      try {
+        this.displayOptions[key] = JSON.parse(value);
+      } catch (_) {
+        console.log(
+          'Not expected value for key "',
```

Comment:
Can you please use a template string here?

Reply:
fixed